packages/@react-stately/datepicker/src/useDateFieldState.ts

```diff
@@ -118,9 +118,13 @@ const PAGE_STEP = {
   second: 15
 };
 
-// Node seems to convert everything to lowercase...
 const TYPE_MAPPING = {
-  dayperiod: 'dayPeriod'
+  // Node seems to convert everything to lowercase...
```

Comment:
Are you trying on node 22? I'm not seeing the issue in my local node environment
![Screenshot 2025-05-22 at 5 40 14 pm](https://github.com/user-attachments/assets/d98f0405-6c50-4014-b4cf-00ba5500e994)

Reply:
this was there before. probably from some older version